requirements/base.in

```diff
@@ -0,0 +1,4 @@
+# Core requirements for using this package
+-c constraints.txt
+
+Click>=6.0
```

Comment:
Don't use the version here just package name is enough. Use constraint.txt for this purpose.